docs/playbook/ingesting-data/005-ingesting-dynamo-db-tables.md

```diff
@@ -0,0 +1,178 @@
+The Dynamo DB instance must be in an AWS account that is linked to the Hackney Hub.
+
+## Intended audience
+
+- Data Engineers
+- Data Analysts
+
+## Prerequisites
+
+- You have a Github account - you can [create one][github_signup] yourself using your Hackney email
+- You have been added to the 'LBHackney-IT' team - you can request this from Rashmi Shetty
+- You have an engineer in the account where the Dynamo DB instance is who can create an IAM role for you in that account.
+
+
+## Creating a role in the account where the Dynamo DB instance is
+
+You will need someone who has access to the account where the Dynamo DB tables live to help with this.
+They will need to create two roles, for both our pre-production and production AWS accounts.
+The roles will both need to have a trust policy that looks like this. One will hold the pre-production account number and one the production account number.
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Principal": {
+                "AWS": "<account number of data platform account>"
+            }
+        }
+    ]
+}
+```
+Both roles should have the following policy attached to the role.
+Where ` -- TABLE ARNs HERE -- ` can be replaces by a list of ARNs for the tables that you want to ingest. 
+```
+{
+  "Version": "2012-10-17"
+  "Statement" [
+    {
+      "Effect" = "Allow"
+      "Action" = [
+        "dynamodb:List*",
+        "dynamodb:Get*",
+        "dynamodb:Query",
+        "dynamodb:Scan",
+        "dynamodb:BatchGet*",
+        "dynamodb:DescribeTable*",
+        "dynamodb:DescribeStream*",
+      ]
+      "Resource" = [
+        -- TABLE ARNs HERE --
+      ]
+    }
+  ]
+}
+```
+
+You will need the role ARN's of both roles, once they have been created, for the next step.
+
+## Saving the role ARN in the data platform accounts
+
+Complete this step in both pre-production & production AWS accounts.
+
+1. Login to AWS via [Hackneys SSO][hackney_sso].
+1. Navigate to [SSM parameter store][aws_console_ssm] in the AWS console.
+1. Click "Create parameter".
+1. Set the name of the parmeter to be something like `/where-the-data-is-coming-from/prod/role-arn-to-access-dynamodb-tables`.
+1. Keep note of the parameter name, as you will need it in the next section, and click "Create parameter".
+
+## Setting up the AWS Glue job
+
+1. Open the [Data Platform Project](https://github.com/LBHackney-IT/data-platform). If you don't have the correct permissions, you'll get a '404' error (see [prerequisites](#prerequisites)).
+2. Navigate to the main `terraform` directory and add a new file named after the dataset you are ingesting, for example `30-mtfh-tables-ingestion.tf`. Click on "Add file" then "Create new file" to do this.
+3. Load the role ARN saved in the [previous section](#saving-the-role-arn-in-the-data-platform-accounts) in to terraform.
+  - Copy this example code block into the file.
```

Comment:
Took me a moment to realise that the example code block starts on line 90. Could the block be moved to just below this bullet point, instead of at the bottom of all three?